app/Services/VersionService.php

```diff
@@ -17,16 +19,19 @@
  */
 class VersionService extends ServiceProvider
 {
+    private const VERSIONS_TO_KEEP = 3;
```

Comment:
Potential candidate for a .env variable? Unlikely we'd ever change I guess

Reply:
If anything, more likely to be removed entirely!